build.gradle

```diff
@@ -0,0 +1,103 @@
+plugins {
+    id 'java'
+    id 'eu.xenit.alfresco' version "0.2.0"
+    id 'eu.xenit.amp' version "0.2.0"
+    id "eu.xenit.docker-alfresco" version "4.0.2"
+}
+
+description = 'Remote-jodconverter for Alfresco'
+
+group = 'eu.xenit.transformers'
+version = '1.1.5-SNAPSHOT'
+
+sourceCompatibility = '1.8'
+
+repositories {
+    mavenLocal()
+}
+
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+ext.alfrescoVersion = "5.1.g"
+
+dependencies {
+    alfrescoProvided "org.alfresco:alfresco-repository:${alfrescoVersion}"
+
+    /* Alfresco with remote jod converter amp docker image */
+    alfrescoAmp files(project.amp)
+    baseAlfrescoWar "org.alfresco:alfresco:${alfrescoVersion}@war"
+    baseShareWar "org.alfresco:share:${alfrescoVersion}@war"    
+}
+
+
+dockerAlfresco {
+    baseImage = "hub.xenit.eu/alfresco-community:${alfrescoVersion}"
+    // current base images have already a version of jodconverter amp, therefore delete that first (leanImage=false)
+    leanImage = false
+
+    dockerBuild {
+        repository = 'xeniteu/alfresco-remote-jodconverter'
```

Comment:
We don't want to push any alfresco image, so no need to give a full repository here (you can just call it `alfresco-remote-jodconverter`)